### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by zencatalyst**
Fixed grammar and spelling issues

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,15 +61,15 @@ variable or `make` parameter), and will soon be moving to v18.
...
+you'd run `make EXTRA_ARGS='--language c++'`. The `Makefile` will automatically install all the third-party libraries
+needed to run; using `npm` to install server-side and client-side components.
 
 For development, we suggest using `make dev` to enable some useful features, such as automatic reloading on file changes
 and shorter startup times.
 
 You can also use `npm run dev` to run if `make dev` doesn't work on your machine.
 
-Some languages need extra tools to demangle them, e.g. `rust`, `d`, or `haskell`. Such tools are kept separately in the
+Some languages need extra tools to demangle them, e.g. `rust`, `d`, or `Haskell`. Such tools are kept separately in the
```
Comment (partouf) [inline README.md:72]: this seems arbitrary, technically all these languages start with a capital letter

Comment (zencatalyst) [reply README.md:72]: Could you please fix it? Or should I do it myself?

Comment (zencatalyst) [reply README.md:72]: > this seems arbitrary, technically all these languages start with a capital letter

Could you please fix it? Or should I do it myself?

File: README.md [hunk 2]
```diff
@@ -6,17 +6,16 @@
 # Compiler Explorer
 
 **Compiler Explorer** is an interactive compiler exploration website. Edit code in C, C++, C#, F#, Rust, Go, D, Haskell,
-Swift, Pascal, [ispc](https://ispc.github.io/), Python, Java or in any of the other
-[30+ supported languages](https://godbolt.org/api/languages), and see how that code looks after being compiled in real
-time. Multiple compilers are supported for each language, many different tools and visualisations are available, and the
+Swift, Pascal, [ispc](https://ispc.github.io/), Python, Java, or any of the other
+[30+ supported languages](https://godbolt.org/api/languages), and see how that code looks after being compiled in real time. Multiple compilers are supported for each language, many different tools and visualizations are available, and the
```
Comment (mattgodbolt) [inline README.md:10]: Can you keep to the same line limit as the other lines here. I thought we had a pre-commit hook for this...sorry it didn't report anything.

File: README.md [hunk 3]
```diff
@@ -6,17 +6,16 @@
...
 UI layout is configurable (thanks to [GoldenLayout](https://www.golden-layout.com/)).
 
 Try out at [godbolt.org](https://godbolt.org), or [run your own local instance](#running-a-local-instance).
 
 **Compiler Explorer** follows a [Code of Conduct](CODE_OF_CONDUCT.md) which aims to foster an open and welcoming
 environment.
 
-**Compiler Explorer** was started in 2012 to show how C++ constructs translated to assembly code. It started out as a
+**Compiler Explorer** was started in 2012 to show how C++ constructs are translated to assembly code. It started as a
```
Comment (mattgodbolt) [inline README.md:18]: I don't know that this materially helps the understanding but if you feel this is simpler to understand...

File: README.md [hunk 4]
```diff
@@ -33,17 +32,17 @@ or by buying cool gear on the [Compiler Explorer store](https://shop.spreadshirt
 
 There is now a FAQ section [in the repository wiki](https://github.com/compiler-explorer/compiler-explorer/wiki/FAQ). If
 your question is not present, please contact us as described below, so we can help you. If you find that the FAQ is
-lacking some important point, please free to contribute to it and/or ask us to clarify it.
+lacking some important point, please feel free to contribute to it and/or ask us to clarify it.
```
Comment (mattgodbolt) [inline README.md:35]: :+1:

File: README.md [hunk 5]
```diff
@@ -33,17 +32,17 @@ or by buying cool gear on the [Compiler Explorer store](https://shop.spreadshirt
...
 
 ### Videos
 
-There are a number of videos that showcase some features of Compiler Explorer:
+Several videos showcase some features of Compiler Explorer:
```
Comment (mattgodbolt) [inline README.md:39]: :+1:

File: README.md (hunk 1 above)
Comment (mattgodbolt) [reply README.md:72]: Can you please fix it

**Review by mattgodbolt — CHANGES_REQUESTED** [review]
Thanks for this - I mostly agree with the changes. Can you make the small changes needed and we'll merge it in!

**Comment by zencatalyst** [comment]
@mattgodbolt I think I have fixed the issues. Please see if it's ready to be merged with the main.

**Comment by zencatalyst** [comment]
@partouf Please review the changes.

**Comment by partouf** [comment]
thanks!

**Comment by mattgodbolt** [comment]
Thanks both

**Pull request merged**